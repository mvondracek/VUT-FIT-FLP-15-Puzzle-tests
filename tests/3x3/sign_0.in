-100 -5 0
0 1 5
* 100 999
