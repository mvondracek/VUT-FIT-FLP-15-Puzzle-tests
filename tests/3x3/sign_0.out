-100 -5 0
0 1 5
* 100 999

-100 -5 0
0 1 5
100 * 999

-100 -5 0
0 1 5
100 999 *
